.github/workflows/ci.yaml

```diff
@@ -85,6 +85,10 @@ jobs:
     - name: Compile
       run: cargo test --no-run
 
+    # We have to build rust-analyzer first for running related heavy tests
+    - name: Build rust-analyzer
+      run: cargo build -p rust-analyzer
```

Comment:
I think this should not be neccessary -- cagro guarantees that the binaries are build before integration tests .

Reply:
Okay, removed,